docs/user-guide/cli-authentication-methods.md

```diff
@@ -0,0 +1,288 @@
+
+    A profile with a port number:
+
+    ```json
+        "type": "zosmf",
+        "properties": {
+            // highlight-start
+            "port": 443
+            // highlight-end
+        }
+    ```
+
+    A profile with a base path:
+
+    ```json
+    "type": "zosmf",
+        "properties": {
+              // highlight-start
+            "basePath": "/ibmzosmf/api/v1"
+              // highlight-end
+        }
+    ```
+
+     If you use the  `--show-token` option with the `login` command in Step 1, you must manually supply the token on each command using the `--token-value` option. For example:
+
+     ```
+     zowe zos-files list data-set <searchPattern> --token-value <123>
+     ```
+    - `data-set <searchPattern>`
+
+        Specifies the data set search criteria (for example, `<IBMUSER.*>`)
+    - `--token-value <123>`
+
+        Specifies the token value supplied in Step 2.
+
+:::note notes
+
+* Tokens expire after a period of time defined by your security administrator. When a token expires, you must log in to API ML again to get a new token.
+* If you omit connection details from a service profile, such as a `zosmf` profile, the CLI uses the information from your base profile.
+* You can choose to specify all connection details on a service profile and connect directly to the service. Routing through API ML is not required.
+
+:::
+
+### Logging in with a client certificate
+
+Use a client certificate to generate a token and log in to API ML:
+
+```
+zowe auth login apiml --host <APIML Host> --port <APIML Port> --cert-file <PEM Public Certificate Path> --cert-key-file <PEM Private Key File Path>
+```
+
+- `--host <APIML Host>`
+
+    Specifies the API ML host.
+- `--port <APIML Port>`
+    
+    Specifies the API ML port.
+- `--cert-file <PEM Public Certificate Path>`
+
+    Specifies the path for the PEM public certificate.
+- `--cert-key-file <PEM Private Key File Path>`
+
+    Specifies the path to the PEM private certificate.
+
+Zowe CLI obtains a security token from API ML and adds that token to the base profile in the applicable configuration file.
+
+   ```json
+         "project_base": {
+            "type": "base",
+            "properties": {
+                  // highlight-start
+                "tokenType": "apimlAuthenticationToken",
+                   // highlight-end
+                ...
+            },
+            "secure": [
+                  // highlight-start
+                "tokenValue"
+                   // highlight-end
+            ]
+        }
+   ``` 
+
+:::note
+
+If you have multiple types of configuration files and base profiles, see [How configuration files and profiles work together](../user-guide/cli-using-understand-profiles-configs.md#how-configuration-files-and-profiles-work-together) to learn which configuration and profile would be used to store the API ML token.
```

Comment:
We could also suggest here to see [Specifying a base profile](https://docs.zowe.org/stable/user-guide/cli-using-integrating-apiml/#specifying-a-base-profile) to learn how log in/out of a different base profile.